src/main/java/application/controller/LoginController.java

```diff
@@ -0,0 +1,45 @@
+package application.controller;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import session.SessionManager;
+import webserver.RequestHandler;
+import webserver.annotation.Controller;
+import webserver.annotation.RequestMapping;
+import application.db.Database;
+import application.model.User;
+import http.request.HttpRequest;
+import http.response.HttpResponse;
+import util.HttpMethod;
+
+import java.util.Map;
+
+@Controller
+public class LoginController extends FrontController {
+    private static final Logger log = LoggerFactory.getLogger(RequestHandler.class);
+
+    @RequestMapping(path = "/users/login", method = HttpMethod.POST)
+    public String login(HttpRequest request
+            , HttpResponse response) {
+        Map<String, String> parameters = request.getHttpRequestBody().getBody();
+        log.debug("userId : {}", parameters.get("userId"));
```

Comment:
```suggestion
        log.debug("userId : {}", request.getParameter("userId"));
```
reqeust에서 map을 생성해서 리턴하는 것보다는 parameter를 가져올 수 있는 메소드가 좋습니다.